charts/gsp-cluster/values.yaml

```diff
@@ -402,13 +402,20 @@ concourse:
     additionalVolumeMounts:
     - name: ci-web-configuration
       mountPath: /web-configuration
+    resources:
+      requests:
+        cpu: 100m
+        memory: 1Gi
   monitor:
     create: true
   worker:
     nameOverride: concourse-worker
     replicas: 2
     hardAntiAffinity: true
-    resources: {}
```

Comment:
I think this was by desire. We basically said, that our `worker`s have all the resources they want. This due to the fact that they live in their own ec2 instances. Not sure if tying them up is a desired behaviour 🤔

Reply:
`requests` are a minimum. I haven't set any `limits`, which would set a maximum, so they absolutely can have all the resources they want.

Note that there is already a `requests` defined by the helm chart, it's just laughably low.

Reply:
My bad. Yes.